main.tf

```diff
@@ -4,7 +4,7 @@ data "aws_region" "default" {}
 
 # Define composite variables for resources
 module "label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.1"
+  source     = "github.com/cloudposse/terraform-null-label.git?ref=master"
   namespace  = "${var.namespace}"
   name       = "${var.name}"
   stage      = "${var.stage}"
```

Comment:
please use tagged releases for stability
the latest version is `ref=tags/0.3.3`

Reply:
sorry, just released https://github.com/cloudposse/terraform-terraform-label/releases/tag/0.1.0
please use `ref=tags/0.1.0` here

Reply:
Done